### DISCUSSION TITLE
Bankable CC14Potentiometer

### DISCUSSION THREAD

**Discussion by officialpalash** [tttapa/Control-Surface#763] [Q&A]
Bankable CC14Potentiometer
Please add this feature or 
Any shortcut way

**Comment by Veroledez** [comment]
La funcion bank ya esta implementada para todos los elementos midi

**Reply by officialpalash** [reply]
![Capture](https://user-images.githubusercontent.com/51896342/174724911-7140f18d-437b-4fd7-a01d-e40336f131f6.PNG)

**Reply by officialpalash** [reply]
Where im doing wrong

**Comment by officialpalash** [comment]
![Capture](https://user-images.githubusercontent.com/51896342/174721330-a6b34336-0a8c-48d2-875f-1b8a96f003ca.PNG)

**Reply by officialpalash** [reply]
Where i am doing wrong

**Reply by Veroledez** [reply]
Cc14potentiometer no esta definido por la libreria,  usa CCPotentiometer, te debe aparecer en letras rojas

**Reply by officialpalash** [reply]
But i need 14 bit cc

**Comment by Veroledez** [comment]
No se si este ejemplo sirva, https://github.com/tttapa/Control-Surface/blob/98c82b463fd3d73b565f0397e84f0bdafc3291ba/examples/Other/GitHub-issues/One-Pot-Both-PB-and-CC/One-Pot-Both-PB-and-CC.ino

Quita el no_channel y pon el que necesitas
